docs/data/sdks/typescript-ampli.md

```diff
@@ -0,0 +1,339 @@
+The `load()` function accepts an options object to configure the SDK's behavior:
+
+| <div class ="big-column">Option</div> | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|---------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `disabled`                            | Optional. Boolean. Specifies whether the Ampli SDK does any work. When `true`, all calls to the Ampli SDK are no-ops. Useful in local or development environments.<br /><br />Defaults to `false`.                                                                                                                                                   |
+| `environment`                         | Optional. String. Specifies the environment the Ampli SDK is running in: `production` or `development`.<br /><br />Environment determines which Access Token is used to load the underlying analytics provider libraries.<br /><br />Defaults to `development`.                                                                                                                                                                                                                    |
+| `client.apiKey`                       |Optional. String. Specifies an API Key. This option overrides the default, which is the API Key configured in your tracking plan.|
+| `client.instance`                     | Optional. AmpltitudeClient. Specifies an Amplitude instance. By default Ampli creates an instance for you.|
+| `client.options`                      | Optional. Amplitude.Config. Overrides the default configuration for the AmplitudeClient.|
+
```

Comment:
This should be client.configuration. The description looks good.
https://github.com/amplitude/ampli-examples/blob/main/python/django/v1/ampli_app/ampli/ampli.py#L55